### PR TITLE
Update Method Name

### PR DESCRIPTION
**Pull request by clxmente**
Fixes the build error

### CODE REVIEW

File: UnknownWorldTraits/mod_info.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 supportedContent: VANILLA_ID,EXPANSION1_ID
-minimumSupportedBuild: 469369
+minimumSupportedBuild: 600112
```
Comment (PsychoDad9999) [inline UnknownWorldTraits/mod_info.yaml]: Should be 596100. Method name was changed in U51

File: UnknownWorldTraits/SimpleInfoScreen_Patch.cs [hunk 2]
```diff
@@ -13,7 +13,7 @@
 namespace OniMods.UnknownWorldTraits
 {
     [HarmonyPatch(typeof(SimpleInfoScreen))]
-    [HarmonyPatch("RefreshWorld")]
+    [HarmonyPatch("RefreshWorldPanel")]
```
Comment (PsychoDad9999) [inline UnknownWorldTraits/SimpleInfoScreen_Patch.cs:16]: looks good to me

**Review by PsychoDad9999 — CHANGES_REQUESTED** [review]
Please change the minimumSupportedBuild to 596100

**Review by clxmente — COMMENTED** [review]
Updated minimumSupportedBuild

**Comment by clxmente** [comment]
Out of curiosity, where did you see/find in what update the method was updated? I was trying to find that information but I don't play ONI so I didn't really know where to look  🥲

**Review by PsychoDad9999 — APPROVED** [review]

**Comment by PsychoDad9999** [comment]
> Out of curiosity, where did you see/find in what update the method was updated? I was trying to find that information but I don't play ONI so I didn't really know where to look 🥲

I checked it myself with ILSpy by reverting the game version in steam. First reverted to the last version of Update 50 (U50-587362) by selecting the beta branch in steam. This version uses the old method name. After that I updated to the first release version of Update 51 (U51-596100) by using the download_depot command in the steam console and this version uses the new method name.

**Pull request merged**